modules/stanford_news_extras/stanford_news_extras.install

```diff
@@ -0,0 +1,42 @@
+<?php
+
+
+/**
+ * Implements hook_install().
+ */
+function stanford_news_extras_install() {
+
+  // Get the default banner image
+  $filename = "news-extra-default-image.png";
```

Comment:
Please namespace this image with the whole module name in order to avoid a possible overwrite of an existing image.